datasets/gbif.yaml

```diff
@@ -15,7 +15,8 @@ Tags:
   - bioinformatics
   - conservation
   - life sciences
-License: "This dataset is available under a [CC-BY](https://creativecommons.org/licenses/by/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset."
+License: "This dataset is available under a [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) license and with the GBIF [terms of use](https://www.gbif.org/terms). Please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines) when using this dataset and include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots)."
+Citation: "`DOI` (Please include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots))"
```

Comment:
@timrobertson100 thanks for the ping! So it looks like there are citation guidelines in the license. Knowing also that the Citation field will be appended to the default language citing the data retrieval from Registry of Open Data, what if we rephrased Citation field as:

In addition, include the relevant DOI found from [the table](https://www.gbif.org/occurrence-snapshots)). For more information on how to cite GBIF datasets, please refer to the GBIF [citation guidelines](https://www.gbif.org/citation-guidelines).

Reply:
@timrobertson100 let me know what you think of these suggestions!

Erin

Reply:
Thanks for the suggestions @chueatwork - I've applied these in a subsequent commit.